test/Microsoft.AspNet.Mvc.IntegrationTests/ByteArrayModelBinderIntegrationTest.cs

```diff
@@ -76,7 +76,7 @@ private class Person
             Assert.Equal(ModelValidationState.Valid, modelState[key].ValidationState); // Should be skipped. bug#2447
         }
 
-        [Fact]
+        [Fact(Skip = "ByteArrayModelBinder should return a non-null result #2456")]
         public async Task BindParameter_NoData_DoesNotGetBound()
```

Comment:
I marked this one as failing... because it's passing for the wrong reason. The verification steps in the test code are wrong, and after correcting them it will pass because `GenericModelBinder` results in the same result.

I still put in the 'skip' here to have a reminder that we have work to do.